Resources/Prototypes/_CM14/Roles/Jobs/Medical/chief_medical_officer.yml

```
@@ -46,7 +46,7 @@
   equipment:
     jumpsuit: CMJumpsuitCMO
     outerClothing: CMCoatLong
-    back: CMSatchelMedical
+    back: CMSatchel
```

This is the leather satchel I think, the marine one is CMSatchelMarine
```suggestion
    back: CMSatchelMarine
```